src/persistence.spec.ts

```diff
@@ -221,7 +222,7 @@ describe("Persistence", () => {
         },
         body: {
           encoding: "base64",
-          data: "GxcAAI6UrMm1WkAERl0HoDFuCn3CIekc",
+          data: "GxcA+I+UrMm1WkAERl0HoDFuCn3CAZLOAQ==",
```

Comment:
The brotli encoding default compression configuration differs between the old library and the new NodeJS builtin implementation. This is why hard-coded expected encoding here differs.